lib/promoted/ruby/client.rb

```diff
@@ -5,67 +5,103 @@
 module Promoted
   module Ruby
     module Client
-      class Error < StandardError; end
-      attr_accessor :options
-      BASE_URL          = "http://wh12.lvh.me:3000"
-      DELIVERY_ENDPOINT = "#{BASE_URL}/deliver"
-      LOGGING_ENDPOINT  =  "#{BASE_URL}/log_request"
-
-      def self.send_request payload, endpoint=nil
-        endpoint ||= BASE_URL
-        response = Faraday.post(endpoint) do |req|
-          req.headers['Content-Type'] = 'application/json'
-          req.body                    = payload.to_json
+
+      DEFAULT_DELIVERY_TIMEOUT_MILLIS = 3000
+      DEFAULT_METRICS_TIMEOUT_MILLIS = 250
+
+      class PromotedClient
```

Comment:
side-comment - Feel free to update the author field in this package.  We want follow up comments and stuff to go to you or the company.

Reply:
Done.